src/istio/ambient/chart/templates/exemptions.yaml

```
@@ -0,0 +1,39 @@
+# Copyright 2024 Defense Unicorns
+# SPDX-License-Identifier: AGPL-3.0-or-later OR LicenseRef-Defense-Unicorns-Commercial
+
+apiVersion: uds.dev/v1alpha1
+kind: Exemption
+metadata:
+ name: istio
+ namespace: uds-policy-exemptions
+spec:
+ exemptions:
+   - policies:
+       - DisallowPrivileged # CNI plugin requires NET_ADMIN, NET_RAW, SYS_PTRACE, SYS_ADMIN, and DAC_OVERRIDE capabilities
+       - RequireNonRootUser # CNI plugin requires root
+       - RestrictVolumeTypes # CNI plugin uses 'hostPath' volume type
+       - DisallowHostNamespaces # CNI plugin requires host namespace access for networking
+       - DropAllCapabilities # CNI plugin requires NET_ADMIN, NET_RAW, SYS_PTRACE, SYS_ADMIN, and DAC_OVERRIDE capabilities
+       - RestrictCapabilities # CNI plugin requires NET_ADMIN, NET_RAW, SYS_PTRACE, SYS_ADMIN, and DAC_OVERRIDE capabilities
+       - RestrictHostPathWrite # CNI plugin requires access to write to CNI
+       - RestrictHostPorts # CNI plugin requires access to host ports
+     matcher:
+       namespace: istio-system
+       kind: pod
+       name: "istio-cni-node.*"
+     title: "Istio CNI exemptions"
+     description: "Exemptions necessary for Istio CNI to manage network configurations and ensure proper traffic routing"
+   - policies:
+       - DisallowPrivileged # ztunnel requires NET_ADMIN, NET_RAW, and SYS_ADMIN capabilities
+       - RequireNonRootUser # ztunnel requires root
+       - RestrictVolumeTypes # ztunnel uses 'hostPath' volume type
+       - DisallowHostNamespaces # ztunnel requires host namespace access for networking
+       - DropAllCapabilities # ztunnel requires NET_ADMIN, NET_RAW, SYS_PTRACE, SYS_ADMIN, and DAC_OVERRIDE capabilities
+       - RestrictCapabilities # ztunnel requires NET_ADMIN, NET_RAW, SYS_PTRACE, SYS_ADMIN, and DAC_OVERRIDE capabilities
+       - RestrictHostPathWrite # ztunnel requires access to write to CNI
```

Similar comment as above, I didn't need all of these. For ref the securityContext:
```yaml
    securityContext:
      allowPrivilegeEscalation: true
      capabilities:
        add:
        - NET_ADMIN
        - SYS_ADMIN
        - NET_RAW
        drop:
        - ALL
      privileged: false
      readOnlyRootFilesystem: true
      runAsGroup: 1337
      runAsNonRoot: false
      runAsUser: 0
```
I was able to make the following change:
```suggestion
       - DisallowPrivileged # ztunnel runs with allowPrivilegeEscalation
       - RequireNonRootUser # ztunnel requires root
       - RestrictVolumeTypes # ztunnel uses 'hostPath' volume type
       - RestrictCapabilities # ztunnel requires NET_ADMIN, NET_RAW, and SYS_ADMIN capabilities
       - RestrictHostPathWrite # ztunnel requires access to write to CNI
```
and the pod still came up.